_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,462 @@
+
+~~~
+crs.area_of_use
+~~~
+{: .language-python}
+~~~
+AreaOfUse(west=0.0, south=0.0, east=6.0, north=84.0, name='Between 0°E and 6°E, northern hemisphere between equator and 84°N, onshore and offshore. Algeria. Andorra. Belgium. Benin. Burkina Faso. Denmark - North Sea. France. Germany - North Sea. Ghana. Luxembourg. Mali. Netherlands. Niger. Nigeria. Norway. Spain. Togo. United Kingdom (UK) - North Sea.')
+~~~
+{: .output}
+
+> ## Exercise: find the unit of the CRS
+> What units are our data in? See if you can find a method to examine this information using `help(crs)` or `dir(crs)`
+>
+> > ## Answers
+> > `crs.axis_info` tells us that our CRS for our raster has two axis and both are in meters.
+> > We could also get this information from the attribute `raster_ams_b9.rio.crs.linear_units`.
+> {: .solution}
+{: .challenge}
+
+### Understanding pyproj CRS Summary
+Let's break down the pieces of the `pyproj` CRS summary. The string contains all of the individual CRS elements that Python or another GIS might need, separated into distinct sections, and datum.
+
+~~~
+<Derived Projected CRS: EPSG:32631>
+Name: WGS 84 / UTM zone 31N
+Axis Info [cartesian]:
+- E[east]: Easting (metre)
+- N[north]: Northing (metre)
+Area of Use:
+- name: Between 0°E and 6°E, northern hemisphere between equator and 84°N, onshore and offshore. Algeria. Andorra. Belgium. Benin. Burkina Faso. Denmark - North Sea. France. Germany - North Sea. Ghana. Luxembourg. Mali. Netherlands. Niger. Nigeria. Norway. Spain. Togo. United Kingdom (UK) - North Sea.
+- bounds: (0.0, 0.0, 6.0, 84.0)
+Coordinate Operation:
+- name: UTM zone 31N
+- method: Transverse Mercator
+Datum: World Geodetic System 1984 ensemble
+- Ellipsoid: WGS 84
+- Prime Meridian: Greenwich
+~~~
+{: .output}
+
+* **Name** of the projection is UTM zone 31N (UTM has 60 zones, each 6-degrees of longitude in width). The underlying datum is WGS84.
+* **Axis Info**: the CRS shows a Cartesian system with two axes, easting and northing, in meter units.
+* **Area of Use**: the projection is used for a particular range of longitudes `0°E to 6°E` in the northern hemisphere (`0.0°N to 84.0°N`)
+* **Coordinate Operation**: the operation to project the coordinates (if it is projected) onto a cartesian (x, y) plane. Transverse Mercator is accurate for areas with longitudinal widths of a few degrees, hence the distinct UTM zones.
+* **Datum**: Details about the datum, or the reference point for coordinates. `WGS 84` and `NAD 1983` are common datums. `NAD 1983` is [set to be replaced in 2022](https://en.wikipedia.org/wiki/Datum_of_2022).
+
+Note that the zone is unique to the UTM projection. Not all CRSs will have a
+zone. Below is a simplified view of US UTM zones. Image source: Chrismurf at English Wikipedia, via [Wikimedia Commons](https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system#/media/File:Utm-zones-USA.svg) (CC-BY).
```

Comment:
Is it interesting to show the EU zone here over the US zone? https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system#/media/File:LA2-Europe-UTM-zones.png (obtained from the same link.)